packages/client/test/cli/cli-rpc.spec.ts

```diff
@@ -4,17 +4,17 @@ import { Client } from 'jayson/promise'
 
 const cliArgs = ['--rpc', '--ws', '--dev', '--transports=rlpx']
 
-const end = (child: ChildProcessWithoutNullStreams, hasEnded: boolean, t: tape.Test) => {
+const end = (child: ChildProcessWithoutNullStreams, hasEnded: boolean, st: tape.Test) => {
   if (hasEnded) return
   hasEnded = true
   child.stdout.removeAllListeners()
   child.stderr.removeAllListeners()
   child.kill('SIGINT')
-  t.end()
+  st.end()
```

Comment:
I am sure you know what you are doing here but just for my understanding: what/how is this changing things? Wouldn't `t` be a local variable and therefore take precedence over the outer `t`? 🤔

(I am especially interested since this seems to be very much a reocurring thing and it's likely important to grasp for future fixes)

Reply:
I made the change in response to [this cli failure](https://github.com/ethereumjs/ethereumjs-monorepo/runs/4214652340?check_suite_focus=true) on the first commit when opening this PR:

```
# [CLI] rpc
# should return valid responses from http and ws endpoints
ok 1 read from WS RPC
# http and ws endpoints should not start when cli args omitted
not ok 2 .end() already called: read from HTTP RPC
```

I assumed there was some overlap on `t` being overwritten with each sub-test suite so I made the change to make the inner test `st`, I think it's a better practice to not overwrite this variable, especially when passing it around to functions to call `end` as we do in that file.

Reply:
Ah, that makes sense, thanks!